styleguide/derek/_partials/solutions/header-mediumIMAC.ejs

```diff
@@ -0,0 +1,22 @@
+<!--Header Medium-->
+<div class="imacBanner medium">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="col-sm-6 imacBanner-textContainer">
+        <h2 class="imacBanner-headline imacBanner-headlineMedium">the cloud is a mystery wrapped in a riddle.<br/>We can unwrap it for you</h2>
+        <span class="line left-top"></span>
+        <span class="line left-bottom"></span>
+        <span class="line right-bottom"></span>
+      </div>
+    </div>
+    <div class="imacBanner-buttonRow">
+      <a class="button imacBanner-button">Read Our Case Studies  <i class="fa fa-angle-right imacBanner-buttonIcon"></i></a>
```

Comment:
Can we copy the styles from `button` into `imacBanner-button` so avoid using 2 classes here?